app/views/waste_exemptions_engine/renews/already_renewed.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="text">
+  <h1 class="heading-large"><%= t(".heading") %></h1>
+
+  <p><%= t(".paragraph_1", reference: @registration.reference) %></p>
+  <p><%= t(".paragraph_2_html", make_a_new_registration_link: make_a_new_registration_link) %></p>
+  <p><%= t(".paragraph_3") %></p>
+  <p>
+    <%= t(".paragraph_4.line_1") %> <br/>
+    <%= t(".paragraph_4.line_2") %> <br/>
+    <a href="mailto:enquiries@environment-agency.gov.uk">enquiries@environment-agency.gov.uk</a>
```

Comment:
Can we store the email in the YAML? This would match what we've done elsewhere :)